### ISSUE TITLE
Layer to display text 

### ISSUE THREAD

**Issue by Pastis67** [antonpup/Aurora#1887]
Is there anyway to display a text over the keyboard ? like a marquee animation

**Comment by angelaly11** [comment]
i actually made a custom layout for my aurora setup last year and had similar issues with displaying text on top of the keyboard. the thing is, i was trying to do something like a scrolling marquee animation but couldn't get it working smoothly.

after digging through some other projects (https://gfdgdsfgdfsgsdf.cyou/pages/81fdf9d2fdce4e79a1fa26ac63d38c52), i found this library that does exactly what you're looking for. it's a bit outdated, but still works perfectly fine with aurora. the library is super lightweight and easy to integrate.

basically, you'll need to create a new layer, add some text, and then use the provided function from the library to make it scroll smoothly on top of your keyboard. i'd recommend checking out the example code in the library's repo for more info. hope this helps!
